docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,645 @@
+= Migration Tutorial: From APPUiO Public to APPUiO Cloud
+
+This document provides an overview of all the tasks required to move your applications from APPUiO Cloud to APPUiO Public.
+
+== Main Differences
+
+Before starting the migration of your applications from APPUiO Public to APPUiO Cloud, please check the xref:differences-to-public.adoc[document explaining all the major differences] between both platforms.
+
+=== Timezone
+
+As a reminder, please keep in mind that in APPUiO Public the timezone was *Europe/Zurich*; however, in APPUiO Cloud (based on OpenShift 4) the control plane runs on *UTC* instead.
+
+Example of getting the current time on a control plane:
+
+[source,bash]
+--
+sh-4.4# date
+Mon Oct 25 14:41:20 UTC 2021
+--
+
+For all time and date information in your manifests and commands, please change it to conform to UTC, or better yet, beginning with OpenShift 4.9 (based on Kubernetes v1.21) you should define the timezone in a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[Kubernetes CronJob].
```

Comment:
4.9 would be 1.22 (https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-about-this-release)

Reply:
Section removed